test/controllers/projects_controller_test.rb

```diff
@@ -16,7 +16,7 @@ class ProjectsControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should create project" do
-    assert_difference('Project.count') do
+    assert_difference("Project.count") do
       post projects_url, params: { project: { brideFirstName: @project.brideFirstName, brideLastName: @project.brideLastName, celebrantFirstName: @project.celebrantFirstName, celebrantLastName: @project.celebrantLastName, celebrantNamePrefix: @project.celebrantNamePrefix, church: @project.church, city: @project.city, groomFirstName: @project.groomFirstName, groomLastName: @project.groomLastName, isSecured: @project.isSecured, liturgy: @project.liturgy, weddingdate: @project.weddingdate } }
     end
 
```

Comment:
Same problem here, change the line below to 

```ruby
assert_redirected_to project_url(Project.order(:created_at).last)
```

Reply:
fixed in commit f350f94